### ISSUE TITLE
Bundle CSS and JS in plugin folder

### ISSUE THREAD

**Issue by distantnative** [distantnative/oembed#1]
Would be great to find a way to include the necessary CSS and JS in the plugin folder. Installations and updates would become much easier.

**Comment by distantnative** [comment]
Found in analytics plugin - maybe a way:
```php
return tpl::load(__DIR__ . DS . 'template.php', compact('analytics_id'));
```

**Comment by distantnative** [comment]
Will be included in the upcoming 2.0.0 (already added to the develop branch but still work-in-progress).
